pulser-core/pulser/devices/_device_datacls.py

```diff
@@ -37,11 +37,15 @@
 
 DIMENSIONS = Literal[2, 3]
 
-ALWAYS_OPTIONAL_PARAMS = (
-    "max_sequence_duration",
-    "max_runs",
-    "dmm_objects",
-    "default_noise_model",
+ALWAYS_OPTIONAL_PARAMS = ("max_sequence_duration", "max_runs")
+OPTIONAL_IN_ABSTR_REPR = tuple(
+    list(ALWAYS_OPTIONAL_PARAMS)
+    + [
+        "dmm_objects",
+        "default_noise_model",
+        "requires_layout",
+        "accepts_new_layouts",
+    ]
 )
 PARAMS_WITH_ABSTR_REPR = ("channel_objects", "channel_ids", "dmm_objects")
```

Comment:
I don't know if it has an impact: Does not "Noise Model" have a an abstract repr as well ?

Reply:
Yeah but the name is misleading. These are actually the parameters with a special format in the abstract representation (eg channel IDs and objects get grouped together in the Device abstract repr)